infra/modules/azure_storage_account/examples/complete/locals.tf

```diff
@@ -8,7 +8,27 @@ locals {
     instance_number = "01"
   }
 
-  project = module.naming_convention.project
+  naming_config = {
+    prefix          = local.environment.prefix
+    environment     = local.environment.env_short
+    location        = local.environment.location
+    domain          = local.environment.domain
+    name            = local.environment.app_name
+    instance_number = tonumber(local.environment.instance_number)
```

Comment:
issue: woulnd't this affect the instance_number of vnets in case of multiple storage accounts? what about keeping instance number in arguments instead?